Add install info to GIS applications (#1723)
CSCfi/csc-user-guide: docs/apps/cloudcompare.md

You are free to use CloudCompare for any purpose, including commercially or for education. 

## Citation

`CloudCompare (version 2.10.3) [GPL software]. (2021). Retrieved from http://www.cloudcompare.org.`

If you used a [CloudCompare plugin](http://www.cloudcompare.org/doc/wiki/index.php?title=Plugins), also cite the plugin authors.

## Acknowledgement

Please acknowledge CSC and Geoportti in your publications, it is important for project continuation and funding reports. As an example, you can write "The authors wish to thank CSC - IT Center for Science, Finland (urn:nbn:fi:research-infras-2016072531) and the Open Geospatial Information Infrastructure for Research (Geoportti, urn:nbn:fi:research-infras-2016072513) for computational resources and support".

## Installation

> Cloudcompare was installed to Puhti via Singularity. The Singularity definition file is available on [CSCs 'singularity-recipes' repository](https://github.com/CSCfi/singularity-recipes/blob/main/cloudcompare/cloudcompare.def).

Reviewer: ```suggestion
CloudCompare was installed to Puhti with Apptainer, using this [CloudCompare Apptainer definition file](https://github.com/CSCfi/singularity-recipes/blob/main/cloudcompare/cloudcompare.def).
```
